src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/FagsakDeltagerSøk.tsx

```diff
@@ -113,6 +118,20 @@ const FagsakDeltagerSøk: React.FC = () => {
                 }
             />
 
+            {toggles[ToggleNavn.brukEndringslogg] && innloggetSaksbehandler && (
+                <Endringslogg
+                    userId={innloggetSaksbehandler.navIdent}
+                    dataFetchingIntervalSeconds={60 * 15}
+                    appId={'BA'}
+                    backendUrl={'/endringslogg'}
+                    dataset={'production'}
```

Comment:
Kan dette være miljøstyrt? Vi kan for eksempel bruke `erProd` for å benytte prod vs preprod? :)

Reply:
... Ser ut som at vi bare har et miljø og det er produksjonsmiljø ...